app/api/tests.py

```diff
@@ -1,3 +1,15 @@
-# from django.test import TestCase
+from django.test import Client, TestCase
 
-# Create your tests here.
+
+class ApiTest(TestCase):
+    """Test for API"""
+
+    def setUp(self):
+        self.client = Client()
+        self.endpoint = '/api'
+
+    def test_api_endpoint_is_live(self):
+        r = self.client.get(self.endpoint, {'query': ''})
```

Comment:
@andela-eisaac: That was shortened purposely because using `response` would result to long line at line 15 going against PEP8 standard.